lib/renderer/init.ts

```diff
@@ -127,8 +122,9 @@ if (nodeIntegration) {
   }
 }
 
+const { preloadScripts } = ipcRendererUtils.invokeSync(IPC_MESSAGES.BROWSER_SANDBOX_LOAD);
```

Comment:
we should create a new IPC handler for this, which does not load the file contents, which we end up ignoring in the renderer anyway.